docs/data-sources/aviatrix_all_network_domains.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_all_network_domain"
+description: |-
+  Get a list of all network domains and detail
```

Comment:
In our official documentation, Network Domain is capitalized. We should probably do the same.